code/controllers/subsystem/objects.dm

```diff
@@ -15,6 +15,7 @@ var/datum/subsystem/objects/SSobj
 	NEW_SS_GLOBAL(SSobj)
 
 /datum/subsystem/objects/Initialize(timeofday, zlevel)
+	setupGenetics()
```

Comment:
Most of these are too long to keep in or are about comments I'm removing. I've added plenty of comments in the code to explain some of the new code.